src/store/adjustVault.ts

```diff
@@ -26,6 +26,7 @@ type VaultToAdjust = {
   metrics: VaultMetrics;
   collateralizationRatio?: Ratio;
   createdByOfferId: string;
+  vaultState?: string;
```

Comment:
I think this can be tightened to the list of possible strings

Reply:
Done